src/devices/bus/mc10/mcx128.cpp

```diff
@@ -43,15 +43,17 @@
 //#define VERBOSE (LOG_GENERAL)
 #include "logmacro.h"
 
+namespace {
 
-ROM_START(mc10_mcx128)
+ROM_START(mcx128)
 	ROM_REGION(0x4000, "eprom", ROMREGION_ERASE00)
-	ROM_LOAD("mcx128bas.rom", 0x0000, 0x4000, CRC(11202e4b) SHA1(36c30d0f198a1bffee88ef29d92f2401447a91f4))
-ROM_END
+	ROM_DEFAULT_BIOS("mc10_mcx128")
 
-ROM_START(alice_mcx128)
-	ROM_REGION(0x4000, "eprom", ROMREGION_ERASE00)
-	ROM_LOAD("alice128bas.rom", 0x0000, 0x4000, CRC(a737544a) SHA1(c8fd92705fc42deb6a0ffac6274e27fd61ecd4cc))
+	ROM_SYSTEM_BIOS(0, "mc10_mcx128", "Darren Atkinson's MCX-128 cartridge")
+	ROMX_LOAD("mcx128bas.rom", 0x0000, 0x4000, CRC(11202e4b) SHA1(36c30d0f198a1bffee88ef29d92f2401447a91f4), ROM_BIOS(0))
+
+	ROM_SYSTEM_BIOS(1, "alice_mcx128", "Darren Atkinson's MCX-128 cartridge for Alice")
+	ROMX_LOAD("alice128bas.rom", 0x0000, 0x4000, CRC(a737544a) SHA1(c8fd92705fc42deb6a0ffac6274e27fd61ecd4cc), ROM_BIOS(1))	
 ROM_END
```

Comment:
Since you know it’s an MCX-128, you can abbreviate the BIOS tags to just `ROM_SYSTEM_BIOS(0, "mc10", "Darren Atkinson's MCX-128 cartridge")` and `ROM_SYSTEM_BIOS(1, "alice", "Darren Atkinson's MCX-128 cartridge for Alice")` to make configuration more terse if you like.